src/TicketSatisfaction.php

```diff
@@ -123,7 +123,8 @@ public function showSatisactionForm($ticket)
 
            // Set default satisfaction to 3 if not set
             if (is_null($this->fields["satisfaction"])) {
-                $this->fields["satisfaction"] = 3;
+                $default_rate = Entity::getUsedConfig('inquest_default_rate', $ticket->fields['entities_id'], 'inquest_default_rate');
```

Comment:
`inquest_default_rate` inheritance should be based on `inquest_config` field.
```suggestion
                $default_rate = Entity::getUsedConfig('inquest_config', $ticket->fields['entities_id'], 'inquest_default_rate');
```